src/test/java/org/assertj/core/error/ShouldBeValidBase64_create_Test.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2020 the original author or authors.
+ */
+package org.assertj.core.error;
+
+import static java.lang.String.format;
+import static org.assertj.core.api.BDDAssertions.then;
+import static org.assertj.core.error.ShouldBeValidBase64.shouldBeValidBase64;
+import static org.assertj.core.presentation.StandardRepresentation.STANDARD_REPRESENTATION;
+
+import org.assertj.core.description.Description;
+import org.assertj.core.internal.TestDescription;
+import org.assertj.core.presentation.Representation;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Tests for <code>{@link ShouldHaveNoSuperclass#create(Description, Representation)}</code>.
+ *
+ * @author Stefano Cordio
+ */
+@DisplayName("ShouldBeValidBase64 create")
+class ShouldBeValidBase64_create_Test {
+
+  @Test
+  void should_create_error_message() {
+    // WHEN
+    String message = shouldBeValidBase64("string").create(new TestDescription("TEST"), STANDARD_REPRESENTATION);
```

Comment:
add a `%` to make sure this does not break a later String.format (this should be fine but we had quite few of those)